src/web/components/SubNav/SubNav.tsx

```diff
@@ -1,87 +1,207 @@
-import React, { Component, createRef } from 'react';
+import React, { useState, useRef, useEffect } from 'react';
+import { css, cx } from 'emotion';
 
 import { GuardianLines } from '@frontend/web/components/GuardianLines';
+import { text } from '@guardian/src-foundations/palette';
+import { palette } from '@guardian/src-foundations';
+import { textSans } from '@guardian/src-foundations/typography';
+import { from } from '@guardian/src-foundations/mq';
+import { pillarPalette, pillarMap } from '@root/src/lib/pillars';
 
-import { Inner } from './Inner';
-
-interface Props {
-    subnav?: SubNavType;
+type Props = {
+    subNavSections?: SubNavType;
```

Comment:
Thanks for using the same name as the property!

This prop no longer needs to be optional as there's now a guard for it in App